app.json

```diff
@@ -289,7 +289,7 @@
         "js_module": "endpoints/kms.js",
         "js_function": "hearthbeat",
         "forwarding_required": "sometimes",
-        "authn_policies": ["user_cert", "member_cert", "jwt"],
+        "authn_policies": [],
```

Comment:
why did you remove this. It provides a nice framework to test authentication

Reply:
I wanted an endpoint for appgateway to check whether service is up or not? Since heartbeat endpoint are generally for this purpose, so I remove any auth from it